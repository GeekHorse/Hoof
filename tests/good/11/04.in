new right one done
new down two done
new in subtwoa done
new in subsubtwo done
new out subtwob done
new in subsubtwob done
save
