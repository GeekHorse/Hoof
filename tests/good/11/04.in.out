new right one done
new down two done
new in subtwoa done
new in subsubtwo done
out
new down subtwob done
new in subsubtwob done
out
out
